src/rdkafka_broker.c

```diff
@@ -3462,9 +3462,21 @@ rd_kafka_fetch_reply_handle (rd_kafka_broker_t *rkb,
 		rd_kafka_op_throttle_time(rkb, rkb->rkb_rk->rk_rep,
 					  Throttle_Time);
 	}
-
+#if WITH_ZSTD
```

Comment:
Remove the #if .. stuff.
These checks are for FetchRequest v10 support, which we'll eventually need for other stuff than zstd.